bin/lhapdf.in

```diff
@@ -277,16 +277,16 @@ def download_url(source, dest_dir, dryrun=False):
             import urllib2 as urllib
         try:
             u = urllib.urlopen(url)
-            file_size = int(u.info().get('Content-Length')[0])
+            #file_size = int(u.info().get('Content-Length')[0])
```

Comment:
```suggestion
            file_size = int(u.info().get('Content-Length', (-1,))[0])
```
wouldn't that be enough and avoid commenting out the other lines (which I'd avoid if possible to keep changes to be ported to a future version to a minimum)